styles/vendor/fotorama/modules/_fullscreen-icon.scss

```diff
@@ -10,12 +10,23 @@
     height: $fotorama__block-size;
     background-image: $fotorama__close-icon;
     background-position: center;
+    background-color: $color-primary;
```

Comment:
not better to use `$fotorama__bg-color` ?

Reply:
yup, good catch, thx